ldk/goja/typedoc.json

```diff
@@ -0,0 +1,25 @@
+{
+    "out": "docs",
+    "includeDeclarations": true,
+    "excludeExternals": true,
+    "excludePrivate": true,
+    "mode": "file",
+    "toc": [],
+    "exclude": [
+      "coverage/**/*",
+      "jest.config.js",
+      "**/*.test.ts"
+    ],
+    "listInvalidSymbolLinks": true,
+    "inputFiles": ["src/index.ts"],
+    "stripInternal": true,
+    "theme": "pages-plugin",
+    "name": "Secure Loop Development Kit for Node.js",
```

Comment:
Should it be node.js or something else?

Reply:
Good question. How about "Javascript Loop Development Kit"?

Reply:
Sure why not ;)